source/runbooks/instance-scheduler-excluding-accounts.html.md.erb

```diff
@@ -0,0 +1,88 @@
+---
+owner_slack: "#modernisation-platform"
+title: Setting Accounts to be excluded from the Instance Scheduler
+last_reviewed_on: 2025-01-15
+review_in: 6 months
+---
+
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-NXTCMQ7ZX6"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', 'G-NXTCMQ7ZX6');
+</script>
+
+# <%= current_page.data.title %>
+
+## Background
+
+There are some Modernisation Platform accounts that should be excluded from the Instance Scheduler: 
+
+- Non-member accounts which do not have the cross-account role & policies added during deployment. Any attempt by the scheduler to access these accounts would fail with an AccessDenied error and as such they are excluded.
+
+- Production member accounts. These are out of the scope of the scheduler to avoid loss of service availability.
+
+- Those non-production member accounts that users have requested are skipped by the scheduler.
+
+## How Instance Scheduler Determines which accounts are in scope
+
+Previously, the Instance Scheduler used a variable containing all of the account names that were to be skipped, including non-member accounts. That method has been changed so instead of a "skip list", this information is obtained form the /environments/*.json files in the Modernisation Platform repository.
```

Comment:
It is probably just worth stating what it currently does - if we need users to take some action, we should probable inform them via the update channel 🆙